cli/cmd/volume.go

```diff
@@ -152,16 +154,44 @@ var volumeResetCmd = &cobra.Command{
 	},
 }
 
+func volumeInfoHandler2(cmd *cobra.Command)(api.Volinfo, error) {
+	var vols api.Volinfo
+	validateNArgs(cmd, 0, 1)
+	volname := ""
+	if len(cmd.Flags().Args()) > 0 {
+		volname = cmd.Flags().Args()[0]
+	}
+	if volname == ""{
+		vols_, err := client.VolumeAll()
+		if err != nil {
+			log.Println("No volumes present")
+			failure(fmt.Sprintf("Error getting Volumes list %s", err.Error()), 1)
+		}
+		fmt.Println(vols_)
+	}else{
+		vols, _ = client.Volumes(volname)
+	}
+	return vols, nil
```

Comment:
Add display functionality here.

if isInfo{
// Add Info display for loop
} else {
// Add list display as table with heading ID  Name
}

Reply:
Done!